tests/unit/core/actions/test_addons.js

```diff
@@ -1,7 +1,8 @@
-import { fetchAddon } from 'core/actions/addons';
+import { fetchAddon } from 'core/reducers/addons';
 import { createStubErrorHandler } from 'tests/unit/helpers';
 
-describe('core.actions.addons', () => {
+
+describe(__filename, () => {
```

Comment:
I think this file should just be merged into [`tests/unit/core/reducers/test_addons.js`](https://github.com/willdurand/addons-frontend/blob/933f81cfd86ece933e6149186fe4a2f168c6b609/tests/unit/core/reducers/test_addons.js).